kernel/arch/dreamcast/hardware/biosfont.c

```diff
@@ -119,15 +122,15 @@ uint32 sjis2jis(uint32 sjis) {
 
 
 /* EUC -> JIS conversion */
-uint32 euc2jis(uint32 euc) {
+static uint32_t euc2jis(uint32_t euc) {
     return euc & ~0x8080;
 }
 
 /* Given an ASCII character, find it in the BIOS font if possible */
-uint8 *bfont_find_char(uint32 ch) {
-    uint8   *fa = syscall_font_address();
+uint8_t *bfont_find_char(uint32_t ch) {
+    uint8_t   *fa = syscall_font_address();
```

Comment:
The font address is static, we can call `syscall_font_address()` once, and not every character.

Reply:
Good point! I'm caching it now.

Reply:
> Good point! I'm caching it now.

It doesn't interfere directly here, but you should keep in mind that the font locking is for access to the data region rather than for the calling of the font_address syscall. Caching the result is a good idea, but abstracts the value from the syscall a bit, so it seemed worthwhile to call that out.